Classic console - Additional Instance Actions - Export / Create instance from snapshot (#613)
neo4j/docs-aura: modules/ROOT/pages/aurads/managing-instances/backup-restore-export.adoc

Restoring a snapshot overwrites the data in your instance, replacing it with the data contained in the snapshot.
====

=== Export

By selecting the ellipses (...) button next to a snapshot, you can *Export*.
*Export* downloads the database as a compressed file, allowing you to store a local copy and work on your data offline.
For AuraDS instances, backups are exported as a `.tar` file because this format bundles both the backup data and the additional metadata required for Graph Data Science.

=== Create instance from snapshot

> *Create instance from snapshot* creates a new AuraDS instance using the data from the snapshot.

Author: You can create an instance using the data from a snapshot, just like creating a clone of an instance.
From the more (*...*) menu next to an existing snapshot select *Create instance from snapshot*.
This opens a window where you can assign a name to the new instance.

Restoring a snapshot overwrites the data in your instance, replacing it with the data contained in the snapshot.
====

> === Export

Reviewer: ```suggestion
=== Export and create
```

> By selecting the ellipses (...) button next to a snapshot, you can *Export*.
> *Export* downloads the database as a compressed file, allowing you to store a local copy and work on your data offline.
> For AuraDS instances, backups are exported as a `.tar` file because this format bundles both the backup data and the additional metadata required for Graph Data Science.

Reviewer: ```suggestion
From the more menu (...) next to an existing snapshot, you can:
* *Export* - Download the database as a *_.tar_* file, allowing you to store a local copy and work on your data offline.
This format bundles both the backup data and the additional metadata required for Graph Data Science.
* *Create an instance from snapshot* - Create a new instance using the data from the snapshot.
```

> === Create instance from snapshot
> 
> You can create an instance using the data from a snapshot, just like creating a clone of an instance.
> From the more (...) menu next to an existing snapshot select *Create instance from snapshot*.
> This opens a window where you can assign a name to the new instance.

Reviewer: ```suggestion
```
